ui/src/components/Comment/index.tsx

```diff
@@ -98,6 +98,9 @@ const Comment = ({ objectId, mode, commentId }) => {
       setComments([...comments, ...data.list]);
     }
     const user: Types.PageUser[] = [];
+    if (data.count <= 3) {
```

Comment:
Should be sorted first and then assigned.

```
if (data.count <= 3) {
      data.list.sort((a, b) => a.created_at - b.created_at);
    }
  if (pageIndex === 1 || pageIndex === 0) {
      setComments(data?.list);
    } else {
      setComments([...comments, ...data.list]);
    }
    
    ... 
```

Reply:
Thank you very much, I have corrected the code.